templates/patient/card/insurance.html.twig

```diff
@@ -15,7 +15,7 @@
 {% for c in ins %}
     <div class="tab-pane {{ loop.first ? "active" }}" id="{{ c.type|attr }}" role="tabpanel" aria-labelledby="{{ c.type|attr }}-tab">
         <div class="text-primary pb-2 pt-1">
-            {{ (c.idOld) ? "Old"|xlt ~ " " ~ c.type|capitalize|text : c.type|capitalize|text }} {{ "Insurance"|xlt }}
+            {{ (c.isOld) ? "Old"|xlt ~ " " ~ c.type|capitalize|text : c.type|capitalize|text }} {{ "Insurance"|xlt }}
```

Comment:
hi @robertdown , can you help me with debugging this twig template please? I thought if I fixed the typo in the property it would solve the display of `Old` but alas...

not sure how to step through twig in a classic debugging manner if that's possible, thanks

Reply:
Do you have your `.env` file set to dev?

Reply:
@stephenwaite See https://www.open-emr.org/wiki/index.php/Twig_Development